js/build.js

```diff
@@ -1,371 +1,362 @@
+            nonInteraction: true
+          });
+
+          $lock.find('.state[data-state=verify]').removeClass('present').addClass('future');
+          _this.calculateElHeight($lock.find('.state[data-state=setup]'));
+          $lock.find('.state[data-state=setup]').removeClass('past').addClass('present');
+          _this.focusOnElement($lock.find('.state[data-state=setup]'));
+          $lock.find('.state[data-state=verify]').find('.num-verify').val('');
+        });
 
-        /**
-         * method used to navigate to another page based on a key of an link action on the configuration
-         * @param redirect_key $input that has the redirect onclick_attribute.
-         */
-        function redirect_to(redirect_key) {
-            Fliplet.Navigate.to(_this.configuration[redirect_key]);
+        $lock.find('.use-touchid').on('click', function() {
+          // GA Track event
+          Fliplet.Analytics.trackEvent({
+            category: 'lockScreen',
+            action: 'touchid_manual_activated'
+          });
+          _this.useTouchId();
+        });
+      },
+      calculateElHeight: function(el) {
+        var elementHeight = el.outerHeight();
+        el.parents('.passcode-wrapper').css('height', elementHeight);
+      },
+      focusOnElement: function(el) {
+        if (Modernizr.android) {
+          setTimeout(function() {
+            el.find('input').focus();
+          }, 500);
         }
+      },
+      initialize_PV: function() {
+        _this.pvName = 'passcode_' + _this.widgetUuid;
+        var dataStructure = {
+          hashedPassCode: false
+        };
 
-        Fliplet.Studio.onEvent(function (event) {
-            if (event.detail.event === 'reload-widget-instance') {
-                setTimeout(function() {
-                    if (_this.passcodePV.hashedPassCode) {
-                        var elementHeight = $('.state[data-state=unlock]').outerHeight();
-                        $('.state[data-state=unlock]').addClass('present');
-                        $('.state[data-state=unlock]').parents('.passcode-wrapper').css('height', elementHeight);
-                        return;
-                    }
-                    var elementHeight = $('.state[data-state=setup]').outerHeight();
-                    $('.state[data-state=setup]').addClass('present');
-                    $('.state[data-state=setup]').parents('.passcode-wrapper').css('height', elementHeight);
+        Fliplet.Security.Storage.create(_this.pvName, dataStructure).then(
+          function(data) {
+            _this.passcodePV = data;
+            if (_this.configuration.hasCustomization) {
+              var event = new CustomEvent(
+                'flLockOnLoadCustomization',
+                {
+                  bubbles: true,
+                  cancelable: true
+                }
+              );
+              document.dispatchEvent(event);
+              return;
+            }
+            _this.initialize_lockScreen_ui();
```

Comment:
Don't use a combination of camel case and underscores. It looks you've simply run a find & replace of `lock_screen` on this file without checking what the result is. Please review the whole file manually so I don't have to pick these out.

```suggestion
            _this.initializeLockScreenUI();
```